server/testutils/containere2e/containere2e.go

```diff
@@ -165,3 +171,28 @@ func NewE2ETestPlugin(t *testing.T, extraOptions ...mmcontainer.MattermostCustom
 
 	return mattermost, store, mockClient, tearDown
 }
+
+func getLoggingConfig() string {
+	return `{
+		"test-console":{
+			"Type": "console",
+			"Format": "plain",
+			"Levels": [
+					{"ID": 5, "Name": "debug", "Stacktrace": false},
+					{"ID": 4, "Name": "info", "Stacktrace": false},
+					{"ID": 3, "Name": "warn", "Stacktrace": false},
+					{"ID": 2, "Name": "error", "Stacktrace": false},
+					{"ID": 1, "Name": "fatal", "Stacktrace": true},
+					{"ID": 0, "Name": "panic", "Stacktrace": true},
+					{"ID": 130, "Name": "RemoteClusterServiceDebug", "Stacktrace": false},
+					{"ID": 131, "Name": "RemoteClusterServiceError", "Stacktrace": true},
+					{"ID": 200, "Name": "SharedChannelServiceDebug", "Stacktrace": false},
+					{"ID": 201, "Name": "SharedChannelServiceError", "Stacktrace": true}
```

Comment:
Should we prepare guidance for SRE to enable these logs in test/production?

Reply:
This was introduced by @wiggin77 in the ce2e tests and I found them very useful for us in that use case, I don't know what do you think about enable that logs in the cloud @wiggin77?

Reply:
They can be helpful.  I can see them not wanting *ServiceDebug but *ServiceError is a good idea.  FWIW, these are enabled for community.